projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.html

```diff
@@ -1,4 +1,9 @@
-<div *ngIf="consignmentStatus.includes(consignment.status)">
+<div
+  *ngIf="
+    config.moduleConfig.consignmentTracking &&
```

Comment:
It is better to move this to order detail page:
<ng-container *ngIf = 'consignmenttrackingEnabled '>
    \<cx-consignment-tracking>\</cx-consignment-tracking>
</ng-container